acceptance/testdata/launcher/Dockerfile.windows

```
@@ -1,7 +1,7 @@
-FROM golang:1.15-nanoserver-1809
+FROM golang:1.16-nanoserver-1809
 
 COPY exec.d/ /go/src/exec.d
-RUN go build -o helper.exe exec.d
+RUN set GO111MODULE=off; go build -o helper.exe exec.d
```

To get this to work, I had to change this:
```suggestion
RUN set "GO111MODULE=off" && go build -o helper.exe exec.d
```

---

Thanks!

---

```suggestion
WORKDIR /go/src
RUN go build -o helper.exe exec.d
```

Like micahyoung said, something is up with the `;` there. It's not necessary anyway because GO111MODULE=off by default on golang:1.16. And because it's off, you can build first switching to the directory (_as opposed to relying on the c:/gopath_)